test/wsdl-parse-test.js

```diff
@@ -14,4 +15,13 @@ describe(__filename, function () {
       done();
     });
   });
+
+  it('should only parse a WSDL once', function(done) {
```

Comment:
Sorry, I don't understand why it would test `require('soap').createClient`, this is related to the WSDL, not `createClient` ?